pkg/discovery/service.go

```diff
@@ -22,11 +22,15 @@ var (
 
 	// Error is a general error class of this package
 	Error = errs.Class("discovery error")
+
+	// Pagination
+	offset = 0
```

Comment:
wait a minute, why is this global?

Reply:
I am under the impression that there is one instance of overlay cache is running per satellite. And the refresh is a periodically ran every 'refresh interval'. Is my understanding right above the single instance of overlay cache (and refresh) per satellite?

Reply:
We wanted it to persist it's place between loops of the refresh function so that each interval isn't biting off a huge chunk of the db. I am putting this on the discovery struct instead of making it global, though.